### DISCUSSION TITLE
如何通过模板新建笔记

### DISCUSSION THREAD

**Discussion by AIR-hl** [windingwind/zotero-better-notes#995] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest Zotero and the latest plugin?

- [X] I have confirmed I'm using the latest Zotero and the latest plugin

### Environment

- OS: 
- Zotero Version: 7.0.0-beta.83+066eda731
- Plugin Version: 1.1.4-beta.77

### Describe the bug

无法通过模板新建笔记，只能新建空白笔记
![image](https://github.com/windingwind/zotero-better-notes/assets/66808901/274c3778-edd2-4c05-ae17-93ede0fb0f62)

**Comment by npj06** [comment]
是的，并且新建的笔记也没办法导入关联文献，只能当作最基础的文档编辑器打字使用

**Comment by windingwind** [comment, marked as answer]
1. 从模板创建笔记
<img width="409" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/ca09ffab-93e2-41e4-a8dc-a1b487b7ff94">

2. 导入模板到笔记
<img width="336" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/1a2ccbf5-eb20-4c7f-afdf-a25b484f9195">

**Reply by npj06** [reply]
感谢大佬
